src/main/java/hudson/plugins/warnings/WarningsDescriptor.java

```diff
@@ -61,6 +56,19 @@ public static String getResultUrl(final String group) {
         }
     }
 
+    /**
+     * Used in Configuration as code plugin
```

Comment:
It can be a comment, but it should not be the description summary. Please add a generic description